spec/beaker/host_prebuilt_steps_spec.rb

```diff
@@ -43,7 +43,7 @@
 
   # Non-cygwin Windows
   it_should_behave_like 'enables_root_login', 'windows', [
-    "sudo su -c \"sed -ri 's/^#?PermitRootLogin no|^#?PermitRootLogin yes/PermitRootLogin yes/' /etc/ssh/sshd_config\""
+    "sed -ri 's/^#?PermitRootLogin /PermitRootLogin yes/' /etc/sshd_config"
```

Comment:
@DavidS it looks like the only thing being tested here is this change. Would you add some form of testing to the other changes so we can detect if we break this in future work?